libs/openchallenges/team/src/lib/team.component.ts

```diff
@@ -48,29 +50,25 @@ export class TeamComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.logo$ = this.imageService.getImage({
-      objectKey: 'openchallenges-icon.svg',
-    });
-    this.thomas$ = this.imageService.getImage({
-      objectKey: 'team/tschaffter.jpeg',
-    });
-    this.rong$ = this.imageService.getImage({
-      objectKey: 'team/rchai.jpeg',
-    });
-    this.verena$ = this.imageService.getImage({
-      objectKey: 'team/vchung.png',
-    });
-    this.maria$ = this.imageService.getImage({
-      objectKey: 'team/mdiaz.png',
-    });
-    this.gaia$ = this.imageService.getImage({
-      objectKey: 'team/gandreoletti.jpeg',
-    });
-    this.jake$ = this.imageService.getImage({
-      objectKey: 'team/jalbrecht.jpeg',
-    });
-    this.sage$ = this.imageService.getImage({
-      objectKey: 'logo/sage-bionetworks-alt.svg',
+    this.logo$ = this.getTeamImage('openchallenges-icon.svg');
+    this.thomas$ = this.getTeamImage('team/tschaffter.jpeg', '500px', '1_1');
```

Comment:
Consider using a more descriptive variable name.

```suggestion
    this.thomasImage$ = this.getTeamImage('team/tschaffter.jpeg', '500px', '1_1');
```